frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/DataFormat/index.tsx

```diff
@@ -1,67 +1,95 @@
 import React, { FC } from "react";
 import { DATASET_ASSET_FORMAT } from "src/common/entities";
-import { FormControl, FormLabel, RadioGroup } from "@mui/material";
-import { InputRadio, Tooltip } from "@czi-sds/components";
-import {
-  TOOLTIP_SLOT_PROPS,
-  TOOLTIP_TITLE,
-} from "src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/DataFormat/constants";
-
+import { FormLabel } from "@mui/material";
+import { FormControl } from "./style";
+import { InputCheckbox } from "@czi-sds/components";
+import { DownloadLinkType } from "src/components/Collections/components/Dataset/components/DownloadDataset/components/Content";
+import { possibleDownloadFormats } from "./constants";
 interface Props {
   handleChange: (format: DATASET_ASSET_FORMAT) => void;
-  isDisabled: boolean;
-  selectedFormat: DATASET_ASSET_FORMAT | "";
-  availableFormats: DATASET_ASSET_FORMAT[];
+  isDisabled?: boolean;
+  selectedFormats: DATASET_ASSET_FORMAT[];
+  availableFormats: Set<DATASET_ASSET_FORMAT>;
+  downloadLinks: DownloadLinkType[];
 }
 
 const DataFormat: FC<Props> = ({
   handleChange: handleChangeRaw,
   isDisabled = false,
-  selectedFormat,
+  selectedFormats,
   availableFormats,
+  downloadLinks,
 }) => {
-  const isH5AD = availableFormats.includes(DATASET_ASSET_FORMAT.H5AD);
-  const isRDS = availableFormats.includes(DATASET_ASSET_FORMAT.RDS);
   const handleChange = (event: React.FormEvent<HTMLElement>) => {
     const value = (event.target as HTMLInputElement)
       .value as DATASET_ASSET_FORMAT;
-
     handleChangeRaw(value);
   };
 
+  const humanFileSize = (size: number) => {
```

Comment:
super nit: Feels like this function can be declared outside of the component (or in a `utils` file and imported) to prevent re-declaration every time the component renders? Same for `getFileSize()`